companies/spreadshirt.json

```
@@ -0,0 +1,22 @@
+{
+    "slug": "spreadshirt",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "commerce"
+    ],
+    "name": "Spreadshirt",
+    "address": "sprd.net AG\nGiesserstr. 27\n04229 Leipzig\nGermany",
```

```suggestion
    "address": "sprd.net AG\nGießerstraße 27\n04229 Leipzig\nGermany",
```

I'd tend to write the full street name here…

---

Yeah, that's better!